app/lib/dags/host_filter.json.jbuilder

```diff
@@ -73,6 +74,16 @@ json.steps do
     additional_files: { star_genome: attr[:star_genome] },
     additional_attributes: { output_gene_file: 'reads_per_gene.star.tab' },
   }
+  if attr[:fastq2]
+    steps << {
+      in: ['star_out'],
+      out: 'insert_size_metrics_out',
```

Comment:
see comments in https://github.com/chanzuckerberg/idseq-dag/pull/215 about naming

Reply:
How about just naming it verbatim after the tool that was used (CollectInsertSizeMetrics)? This has been the convention with STAR, bowtie2, etc for exmaple. That way it's clear where the results come from. Greg would you be comfortable with that?